create-100-selects.js

```diff
@@ -0,0 +1,11 @@
+function create100Selects(selector) {
```

Comment:
Love this! Did you notice that it creates 99 selectors? If you use a less than or equal to 100 it will create the extra one :)
(let i = 1; i <= 100; i++)